crates/api_common/src/request.rs

```diff
@@ -81,16 +82,18 @@ fn html_to_site_metadata(html_bytes: &[u8]) -> Result<SiteMetadata, LemmyError>
     .images
     .get(0)
     .and_then(|ogo| Url::parse(&ogo.url).ok());
-
-  let title = og_title.or(page_title);
-  let description = og_description.or(page_description);
-  let image = og_image;
+  let og_embed_url = page
+    .opengraph
+    .videos
+    .first()
+    .map(|v| Url::parse(&v.url).ok())
+    .flatten();
```

Comment:
Since this is finding the first video, maybe the column name should be `embed_video_url`.

Also I didn't pull down this code to check but hopefully this fails gracefully into a None value.

Reply:
Changed the column name. And just tested it with a link that has no video, works fine.

About clippy, unfortunately it doesnt work locally anymore after upgrading to Rust 1.61 because of `lint `clippy::to_string_in_display` has been renamed to `clippy::recursive_format_impl``. And changing that would break ci i believe.

Reply:
CI is currently broken tho. Maybe you could use rustup to do `rustup install 1.59` , and do `cargo +1.59 fmt`